README.md

```diff
@@ -2,46 +2,134 @@
 
 source code = website
 
-> Build static websites for software projects with zero (currently a fair amount of) configuration.
+> Build a website for your software projects with one command (and zero configuration).
 
-![Logo](https://d30y9cdsu7xlg0.cloudfront.net/png/43478-200.png)
+```
+landr deploy
```

Comment:
Ah, you already did so below. Maybe worth unifying the "installation" and "quick start" sections?